postgres/server/init.sls

```diff
@@ -184,7 +204,7 @@ postgresql-running:
   service.running:
     - name: {{ postgres.service }}
     - enable: True
-   {% if grains.os not in ('MacOS',) %}
+   {% if grains.os not in ('MacOS',) and not db_port %}
```

Comment:
This is the simplest way to reuse the same state but I'm sure it could be improved.  What is the right way to only restart the service when the `port` has been changed?